modules/account-lib/src/coin/baseCoin/baseTransactionBuilder.ts

```diff
@@ -140,6 +140,34 @@ export abstract class BaseTransactionBuilder {
     return this._coinConfig.name;
   }
 
+  /**
+   * Verified validity windows params if them exist and return a valid validity windows.
+   * @param {ValidityWindow} [params] optional validity windows parameter to validate.
+   * @returns {ValidityWindow} verified validity windows or default values
+   */
+  getValidityWindow?(params: ValidityWindow): ValidityWindow {
+    const size = 100000;
+    const window: ValidityWindow = params;
+
+    window.firstValid = window.firstValid ? window.firstValid : 0;
+    window.lastValid = window.lastValid ? window.lastValid : 0;
+    window.minDuration = window.minDuration ? window.minDuration : size;
+    window.maxDuration = window.maxDuration ? window.maxDuration : size;
```

Comment:
I find the prop names `minDuration` and `maxDuration` confusing. If they're always timestamps, doesn't it make more sense to use `firstValid` and `lastValid` when the window is determined by either timestamps or block heights?